nixos/tests/systemd-confinement.nix

```diff
@@ -88,7 +103,7 @@ import ./make-test-python.nix {
         config.confinement.packages = lib.singleton symlink;
         testScript = ''
           with subtest("check if symlinks are properly bind-mounted"):
-              machine.fail("chroot-exec test -e /etc")
+              machine.succeed("chroot-exec rmdir /etc")
```

Comment:
:+1: This actually is better than before, since `machine.fail` could fail on unrelated things.